api/core/core.proto

```diff
@@ -227,7 +186,7 @@ message GetObjectResponse {
 
 message ListObjectsRequest {
     string namespace   = 1;
-    string kind        = 2;
+    Kind   kind        = 2;
```

Comment:
Could we please turn this (both listObjects and getObject) back to string? I really would like to preserve the feature that these APIs can be embedded into other products that their own add additional kinds to list objects, without requiring updating the core APIs.